common/protob/messages-management.proto

```diff
@@ -131,6 +131,7 @@ message Features {
     optional uint32 homescreen_width = 47;         // homescreen width in pixels
     optional uint32 homescreen_height = 48;         // homescreen height in pixels
     optional bool bootloader_locked = 49;       // bootloader is locked
+    optional bool language_version_matches = 50;   // translation blob version matches firmware version
```

Comment:
perhaps even `default=true` to resolve old models that don't send this field?

that way Suite can just `if (!language_version_matches) { sendLanguageBlob(...) }` in exactly the situations when required?
cc @mroz22 what do you think?

Reply:
added in 7abca4e5ec0e4f4072d0d6227cd94b2d6ec90fa4